data/staff.json

```diff
@@ -1,42 +1,82 @@
 [
-  {
-    "user": "Jriwanek",
-    "icon": "https://mcmoddev.com/img/jriwanek.jpg",
-    "role": "Mod Developer | MMD Admin"
+	{
+		"user": "Jriwanek",
+		"icon": "https://mcmoddev.com/img/jriwanek.jpg",
+		"role": "Mod Developer | MMD Admin"
+	},
+	{ 
+		"user": "Darkhax",
+		"icon": "https://mcmoddev.com/img/darkhax.png",
+		"role": "Mod Developer | MMD Admin"
+	},
+	{ 
+		"user": "Insidious",
+		"icon": "https://mcmoddev.com/img/insidious.jpg",
+		"role": "MMD Admin"
+	},
+	{
+		"user": "JamiesWhiteShirt",
+		"icon": "https://mcmoddev.com/img/jamies.jpg",
+		"role": "Moderator"
+	},
+	{
+		"user": "Poke",
+		"icon": "https://mcmoddev.com/img/poke.jpg",
+		"role": "Moderator"
+	},
+	{
+    "user": "DShadowWolf",
+    "icon": "https://mcmoddev.com/img/dshadowwolf.jpg",
+    "role": "Mod Developer | Moderator"
   },
-  { 
-    "user": "Darkhax",
-    "icon": "https://mcmoddev.com/img/darkhax.png",
-    "role": "Mod Developer | MMD Admin"
-  },
-  { 
-    "user": "Insidious",
-    "icon": "https://mcmoddev.com/img/insidious.jpg",
-    "role": "Moderator | MMD Admin"
-  },
-  { 
-    "user": "Cazsius",
-    "icon": "https://mcmoddev.com/img/cazsius.jpg",
-    "role": "Moderator"
-  },
-  { 
-    "user": "Zaweri",
-    "icon": "https://mcmoddev.com/img/zaweri.jpg",
-    "role": "Moderator"
+	{ 
+		"user": "Cazsius",
+		"icon": "https://mcmoddev.com/img/cazsius.jpg",
+		"role": "Mod Developer | Moderator"
+	},
+	{ 
+		"user": "Zaweri",
+		"icon": "https://mcmoddev.com/img/zaweri.jpg",
+		"role": "Moderator | Public Server Admin"
+	},
+	{ 
+		"user": "Jared",
+		"icon": "https://mcmoddev.com/img/jared.jpg",
+		"role": "Dev Server Admin"

```

Comment:
Jared isn't a dev server admin. He is a bot maintainer. Also Diluv.

Reply:
I had Jared for Diluv in my head too, no idea how that ended up like this, fixed.